lib/hanami/generators/template_engine.rb

```diff
@@ -0,0 +1,28 @@
+module Hanami
+  module Generators
+    class TemplateEngine
+
+      SUPPORTED_ENGINES = %w(erb slim haml).freeze
+      DEFAULT_ENGINE = 'erb'.freeze
+
+      attr_reader :engine
```

Comment:
what do you think about `name` instead of `engine`? To me, `template_engine.name`   is more meaningful than `template_engine.engine`